### DISCUSSION TITLE
How to change port?

### DISCUSSION THREAD

**Discussion by jniggemann** [SmilyOrg/photofield#74] [Q&A] [answered]
Using the binary (not docker), is it possible to change the port from 8080 to something else?

**Comment by SmilyOrg** [comment, marked as answer]
Hey there!

If you set the `PHOTOFIELD_ADDRESS` environment variable to e.g. `:8765`, the binary should use it to start on port 8765.

Let me know if that works for you or if you need help setting it up!

**Reply by jniggemann** [reply]
Works like a charm, TY!

**Reply by axiomcity** [reply]
Justed tested with the docker command for another port and it work like a charm,Thanks !
